.github/workflows/frontend-cd.yml

```
@@ -0,0 +1,47 @@
+name: Frontend CD
+
+on:
+  # fe/prod 에 브랜치가 푸시될 때
+  push:
+    branches:
+      - fe/prod
+      - main
```

혹시 모르니까 PR 열릴 때도 트리거 될 수 있도록 추가해보는 건 어떤가요?
추가로 변경된 파일 경로가 `frontend/**`일 때만 실행되도록 `paths`도 설정해두는 것도 좋을 것 같아요!

```suggestion
  pull_request:
    branches: [fe/prod, main]
    paths:
      - "frontend/**"
  
  push:
    branches: [fe/prod, main]
    paths:
      - "frontend/**"
```

---

PR이 열릴 때는 cloudfront에 deploy가 되지 않는게 나을 거 같아 보여요! 프로덕션에 직접 push가 될 때 하는게 괜찮지 않을까요?

---

main으로 바로 진입할 수도 있지 않을까 추가하긴 했는데 제거할게요~

[chore: main 브랜치 제거](https://github.com/woowacourse-teams/2025-coffee-shout/pull/68/commits/35407c71406bd51a918e7fa13b97d4b68b90bb02)

---

감사해용!! fe/dev에 머지하겠습니다~! 🙇🏻‍♀️